CHANGES.md

```diff
@@ -1,5 +1,37 @@
 # Changes
 
+## edge-20.7.2
+
+This edge release moves Linkerd's buindled Prometheus into an add-on. This makes
+the Linkerd Prometheus more configurable, gives it a separate upgrade lifecycle
+from the rest of the control plane, and allows users to disable the bundled
+Prometheus instance. In addition, the Destination service will now use the new
+Kubernetes `EndpointSlice` resource when available. Finally, this release
```

Comment:
Is this actually true? I thought we merged support but it's disabled by configuration.

Reply:
I may have misinterpreted the PR description for #4501. In that case, I'm guessing this should be removed from the changelog, and added in whatever release actually enables `EndpointSlice` support?